bitnami/mongodb/templates/hidden/statefulset.yaml

```diff
@@ -398,11 +398,21 @@ spec:
           {{- end }}
           env:
           {{- if .Values.auth.enabled }}
+          {{- if not (or .Values.metrics.username .Values.metrics.password ) }}
```

Comment:
To avoid confusion we could just check for metrics.username instead. For example, if someone leaves metrics.password with a value but has metrics.username empty, that could cause some issue.

Reply:
I removed `.Values.metrics.password` from the query